go.mod

```diff
@@ -13,10 +13,10 @@ require (
 	golang.org/x/tools v0.31.0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
```

Comment:
Please also update from v0.33.0-beta.0 to v0.33.0 in pkg/applyconfiguration/testdata/cronjob/go.mod

Reply:
Thanks @sbueringer 
Got it.
Fixed